akka-typed/src/main/scala/akka/typed/cluster/Cluster.scala

```diff
@@ -216,6 +213,9 @@ private[akka] final class AdapterClusterImpl(system: ActorSystem[_]) extends Clu
   def isTerminated = adaptedCluster.isTerminated
   def state = adaptedCluster.state
 
+  // TODO arbitrary timeout here because systemActorOf returns future ref and
+  // ActorRef(future-ref) can only work on adapted untyped actors
+  // put it in config instead? or figure out way around futures
   private implicit val timeout: Timeout = 5.seconds
   lazy val subscriptions = Await.result(system.systemActorOf(subscriptionsBehavior(adaptedCluster), "cluster-state-subscriptions"), 5.seconds)
```

Comment:
since we know that it's only for adapted system I think we can do something like:
```
system.toUntyped.systemActorOf(PropsAdapter(subscriptionsBehavior(adaptedCluster)), ...)
```